### ISSUE TITLE
Do Compatability Testing for Other Browsers

### ISSUE THREAD

**Issue by jimjkelly** [jimjkelly/missile-defense#12]
I note in IE on Windows 8 there are compatibility issues, so I'm writing a note to remind myself to do testing.  Scott my assumed targets here are going to be:

Latest Chrome
Latest Firefox
IE 11 and Above
Mobile Safari?
Mobile Chrome?

The mobile browsers will be tough because we'll have to think about how to translate the design without nearly as much screen real estate.

**Comment by wslafoy** [comment]
Latest Chrome and Firefox, IE 11+ are good. Ignore mobile for now, I'd consider those a non-priority for the time being. 

Is microsoft Edge still a thing?

**Comment by jimjkelly** [comment]
Yeah I think it's just what they're calling their latest version of the Browser.
